packit/constants.py

```diff
@@ -71,3 +71,8 @@
 upstream_project_name: {upstream_project_name}
 downstream_package_name: {downstream_package_name}
 """
+SYNCING_NOTE = (
+    "The repository is mantained by packit. "
+    "Because of the sync from upstream to downstream, all changes "
```

Comment:
This is not true: packit service already syncs downstream changes back to upstream. How about removing this sentence and replacing it with just a link to our website `https://packit.dev/`?